examples/aws/main.tf

```diff
@@ -1,5 +1,5 @@
 provider "gardener" {
-  kube_file = "${file("<my-gardener-service-account-kubeconfig>")}"
+  kube_file = "${file("../kubeconfig.yaml")}"

```

Comment:
I would recommend to leave a place holder as the kubeconfig can be at a different location too. So the user can put it himself.